rpc/src/v1/helpers/dispatch.rs

```diff
@@ -491,6 +554,8 @@ impl ProspectiveSigner {
 			},
 			prospective: None,
 			ready: None,
+			post_sign: Some(post_sign),
```

Comment:
It doesn't make sense that `post_sign` is an Option anymore because `poll` assumes that it is always is `Some` otherwise panic and `new` requires it too!

Reply:
the `execute` method on `PostSign` takes `self`, so inorder to take ownership of `post_sign` from `ProspectiveSigner` it needs to be an option. refer to this line

https://github.com/paritytech/parity-ethereum/blob/001a42a98a557edc7c602265a580dd552c871a1d/rpc/src/v1/helpers/dispatch.rs#L592-L599